File: temp/sqlite_inspect.php
<?php

$db = new SQLite3(__DIR__ . '/../database/database.sqlite');

$res = $db->query("SELECT name FROM sqlite_master WHERE type='table' AND name NOT LIKE 'sqlite_%' ORDER BY name;");

while ($row = $res->fetchArray(SQLITE3_ASSOC)) {
    echo $row['name'], PHP_EOL;
}

File: temp/sqlite_inspect.py
import sqlite3
from pathlib import Path

db_path = Path(__file__).resolve().parent.parent / "database" / "database.sqlite"

conn = sqlite3.connect(db_path)

cur = conn.cursor()
cur.execute(
    "SELECT name FROM sqlite_master WHERE type='table' AND name NOT LIKE 'sqlite_%' ORDER BY name"
)

for (name,) in cur.fetchall():
    print(name)

conn.close()
